adapter-http/src/main/resources/data/stargate_token.txt

```diff
@@ -1 +1 @@
-# <<put-token-here>>
\ No newline at end of file
+<<your-token-here>>
```

Comment:
We should probably move this out of the resources, since having a "file not found" error is much more desirable than having a "invalid credentials" issue in the event that they don't have a filesystem version of this file. At least, we should make the names distinctive where this is referenced in basic workloads. Comments? Ideas?

Reply:
👍🏼 to removing this file entirely.